products/java.md

```diff
@@ -12,16 +12,21 @@ releasePolicyLink: https://www.oracle.com/technetwork/java/java-se-support-roadm
 releaseDateColumn: true
 sortReleasesBy: 'releaseCycle'
 releases:
+-   releaseCycle: "19"
+    support: 2023-03-01
+    eol: 2023-03-01
```

Comment:
I think these should probably be `-31`? Seems to be considered "end of March" for previous releases, which makes sense given the support cycle is 6 months, and release is mid-late of the month.

Reply:
@BiNZGi what is your decision about this one ^^ ? with or without this ^^ , i am giving a LGTM so it is  green to merge ( after your decision )

Reply:
I now see it looks like this was discussed earlier and changed to 01 from 31. Not sure this is a good idea, it makes it inconsistent with what we put for previous non-LTS releases.

Reply:
The Oracle doc states "March 2023". I coded with `2023-03-31` because it is the last day of March. captn3m0 reviewed and proposed to change to `03-31` because it's vague. I think we don't have a rule for these cases. I didn't saw that other entries have been created with last day of month. But perhaps this is good rule and we should document it.
What do you think @captn3m0 ?

Reply:
It's probably a bit bike-sheddy, but from my perspective end of month makes most sense as a "placeholder" because the stated support policy for these releases is "6 months", aligned with the feature release cycle. But primarily I feel it'd be good to be consistent - since all of the other entries in the list are currently set to the last day of the given month :-)

If we want to be _even better_, the alternative is that we put the date aligned with the OpenJDK "typical release cadence" which is that they tend to release on "the Tuesdays closest to the 17th", aligned with [Oracle's patch policy](https://www.oracle.com/security-alerts/) (and noted on [Adoptium's support page](https://adoptium.net/support/)). This seems to align with the **actual** release date of both patch and feature releases for the last few years.

The intention is that as soon as, say Java 19 comes out, then Java 18 is no longer supported :-) But that date can be varied and is thus not necessarily 100% known in advance. Relevant text from Oracle's support page
> For the purposes of Oracle Premier Support, non-LTS releases are considered a cumulative set of implementation enhancements of the most recent LTS release. Once a new feature release is made available, any previous non-LTS release will be considered superseded.

So *technically*, Java 18 was EOL/EOSL `2022-09-20`, the same day Java 19 came out, where we currently have it as `2022-09-30`.

Reply:
In any case, I fully support merging this as-is since it addresses other missing info - can always adjust later. Things for updating this 👍